scripts/zones/The_Eldieme_Necropolis/mobs/Namorodo.lua

```diff
@@ -23,14 +23,26 @@ entity.onMobFight = function(mob, target)
 end
 
 entity.onMobDeath = function(mob, player, optParams)
-    local personality = player:getFellowValue("personality")
-    local fellow = player:getFellow()
-    if fellow ~= nil then
-        player:showText(fellow, ID.text.SEEMS_TO_BE_THE_END + xi.fellow_utils.checkPersonality(personality))
-        player:setCharVar("[Quest]Looking_Glass", 3)
-        player:timer(30000, function(playerArg)
-            playerArg:despawnFellow()
-        end) -- 30 sec to talk to fellow
+    if
+        GetMobByID(ID.mob.NAMORODO):isDead() and
+        GetMobByID(ID.mob.NAMORODO + 1):isDead() and
+        GetMobByID(ID.mob.NAMORODO + 2):isDead()
+    then
+        for _, v in ipairs(player:getParty()) do
```

Comment:
onMobDeath is called for every member involved in the fight. 
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/62350957/4537a3b4-4d15-4088-a768-d2bef8b01888)

Reply:
Ah good catch - will add the gate you suggested